src/main/java/org/apache/hadoop/hive/ql/io/avro/AvroGenericRecordReader.java

```diff
@@ -169,6 +168,7 @@ private ZoneId extractWriterTimezoneFromMetadata(JobConf job, FileSplit split,
       }
     } catch (IOException e) {
       // Can't access metadata, carry on.
+      LOG.debug(e.getMessage(), e);
```

Comment:
this is what hive is doing? just log at DEBUG level?

Reply:
yeah - this is clean application of patch

Reply:
I will leave it. Does not matter much.